index.html

```diff
@@ -92,6 +92,16 @@ <h1 class="header">Fish Management</h1>
         <section></section>
       </div> -->
     </div>
+
+    <div class="box closed">
+      <div class="modal">
+        <h2>Nutritional Facts</h2>
+        <div class="nutritional-facts">
+
+        </div>
+        <button class="cancel">CANCEL</button>
```

Comment:
Since there's not really any other actions for the user to perform in this modal, I'd suggest using "Close" instead of "Cancel".